gen3_util/common/__init__.py

```diff
@@ -23,6 +23,30 @@
 
 PROJECT_DIR = '.g3t'
 PROJECT_DIRECTORIES = [PROJECT_DIR, 'META/']  # 'DATA/',
+PROJECT_README = """
+# Data Directory
+
+Welcome to the data directory! This repository contains important data files for our project. Before you proceed, please take note of the following guidelines to ensure the security and integrity of our data.
+
+## Important Note: Do Not Check in Protected Files
+
+Some files in this directory are considered protected and contain sensitive information. **DO NOT** check in or commit these protected files to the version control system (e.g., Git). This is crucial to prevent unauthorized access and to comply with security and privacy policies.
+
+
+## Usage Guidelines:
+
+1. **Read-Only Access:** Unless you have explicit permission to modify or update the data, treat this directory as read-only.
+
+2. **Data Integrity:** Ensure the integrity of the data by following proper procedures for reading, updating, and managing files.
+
+3. **Security Awareness:** Be aware of the sensitivity of the data stored here and take necessary precautions to protect it from unauthorized access.
+
+## How to Obtain Access:
+
+If you need access to these files, please contact the project administrator for access to idp.cbds.ohsu.edu
```

Comment:
My bad, can you remove these lines?  Thanks.

Reply:
see https://github.com/ACED-IDP/gen3_util/pull/63/commits/4ce7639c66878522324603955f7792822cd83417